src/UserContainer.java

```diff
@@ -1,30 +1,23 @@
 import java.util.HashMap;
 import java.util.Map;
-import java.util.Set;
 
 public class UserContainer {
+    private static Map<String, String> users = new HashMap<String, String>();
```

Comment:
Check if it is possible to make it `final`

Reply:
not possible